app/src/main/java/org/p2p/wallet/home/ui/wallet/WalletFragment.kt

```diff
@@ -64,6 +64,7 @@ class WalletFragment :
     private val binding: FragmentWalletBinding by viewBinding()
 
     private val strigaFragmentFactory: StrigaFragmentFactory by inject()
+    private val onOffRampNavigator: OnOffRampNavigator by inject()
```

Comment:
Let's talk about this.

Reply:
What's wrong with this? It's a sort of `fragment factory` but with its own replaceFragment calls to avoid duplications in presenters and fragments